Validation error appears when running docs locally

### Version Information
```json
{
...
  "name": "docs",
  "version": "0.0.0",
...
}
```

### Environment
MacOS - Development

### What is the expected behaviour?
After running `yarn start` local docs development client running on localhost. 

### What is the current behaviour?
When running `yarn start` the following error occurs:
```
$ yarn start
yarn run v1.22.17
$ docusaurus start
[INFO] Starting the development server...
[ERROR] A validation error occurred.
[INFO] The validation system was added recently to Docusaurus as an attempt to avoid user configuration errors.
We may have made some mistakes.
If you think your configuration is valid and should keep working, please open a bug report.
[ERROR] ValidationError: "trackingID" is required
error Command failed with exit code 1.
```

### How to reproduce the issue?
1. Clone https://github.com/ToolJet/ToolJet
2. Open docs folder: `cd ToolJet/docs/`
3. run `yarn install`
4. run `yarn start`


### Please provide any traces or logs that could help here.
Provided in `What is the current behaviour?`

### Any possible solutions?
When running in different than production environment pass `undefined` instead of gtag object in `docusaurus.config.js`

### Can you identify the location in the driver source code where the problem exists?
`docusaurus.config.js`

### If the bug is confirmed, would you be willing to submit a PR?
Yes

--- a/docs/docusaurus.config.js
+++ b/docs/docusaurus.config.js
@@ -1,3 +1,5 @@
+const isProd = process.env.NODE_ENV === 'production';
+
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
   title: 'ToolJet - Documentation',
@@ -112,11 +114,13 @@ module.exports = {
           customCss: require.resolve('./src/css/custom.css'),
         },
         sitemap: {},
-        gtag: {
-          trackingID: process.env.GA_MID,
-          // Optional fields.
-          anonymizeIP: true, // Should IPs be anonymized?
-        },
+        gtag: isProd
+          ? {
+              trackingID: process.env.GA_MID,
+              // Optional fields.
+              anonymizeIP: true, // Should IPs be anonymized?
+            }
+          : undefined,
       },
     ],
   ],

